runners/spark/pom.xml

```diff
@@ -72,7 +72,7 @@
                     <parallel>none</parallel>
                     <failIfNoTests>true</failIfNoTests>
                     <dependenciesToScan>
-                      <dependency>org.apache.beam:java-sdk-all</dependency>
+                      <dependency>org.apache.beam:beam-sdks-java-core</dependency>
```

Comment:
LGTM, this is already fixed in #1055 but if that should go in now, I'll rebase and resolve my PR later.